README.md

```diff
@@ -1,2 +1,7 @@
-# website-template
-A template to create all public facing sites
+Overview
+The goal of this project is to let people integrate their google calendar and join any events according to their schedule. The calendar integration can also help members to schedule meetings with other members of the squad. 
+
+Goals
+1)Add events to google calendar: Real dev squad holds multiple events for its members and as well as for the general public. These events are  listed on the website and one can register for these events by clicking on the link which takes you to the airmeet platform which is used to hold these virtual meetups and one can also add these events in their google calendars. Our goal is to cut down our dependence on the airmeet platform and build our own scheduling system.  
```

Comment:
` airmeet platform which is used to hold these virtual meetups and one can also add these events in their google calendars. Our goal is to cut down our dependence on the airmeet platform and build our own scheduling system.`

This is not the goal, we are free to choose any platform to host our discussion/ session.
Our aim is to provide a platform, currently focussed on integrating technical events or discussions later providing platform to all